vscode/package.json

```diff
@@ -432,12 +432,12 @@
       {
         "command": "qsharp-vscode.setTargetProfile",
         "category": "QDK",
-        "title": "Set the Azure Quantum QIR target profile"
+        "title": "Set the QIR target profile"
       },
       {
         "command": "qsharp-vscode.createNotebook",
         "category": "QDK",
-        "title": "Create an Azure Quantum notebook"
+        "title": "Create a Microsoft Quantum notebook"
```

Comment:
This one creates a sample that _actually_ uses Azure Quantum, so the name may actually be appropriate?